src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +45,16 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+
+        List<Integer> modifiedList = IntStream.range(0, numbers.size())
+                .mapToObj(i -> i % 2 == 1 ? numbers.get(i) - 1 : numbers.get(i))
+                .collect(Collectors.toList());
```

Comment:
YOu can proceed stream opetions. 
dont need to collect here.
```suggestion
                .collect(Collectors.toList());
```